spec/lib/jyson/jyson.nesting.spec.js

```diff
@@ -16,8 +16,8 @@ describe('jyson.nesting.spec: a template with nesting', () => {
   it('must convert an object to "json"', () => {
     const input = {
       a: 1,
-      b: {b: 2},
-      c: {c: {c:3}}
+      b: { b: 2 },
```

Comment:
ugh.. we've changed this back and forth a few times... 

is this just for jyson? or will our other repos have this too?

Reply:
NM. my bad. this is correct

Reply:
all our open source code uses ed's style, all our closed source code uses hubba style. I plan on drying up all our open source code with the same rules.